doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -378,9 +450,13 @@ Declaring Implicit Arguments
        Check (fun l => map length l = map (list nat) nat length l).
 
 .. note::
-   Use the :cmd:`Print Implicit` command to see the implicit arguments
+   Use :cmd:`About` to view the current implicit arguments setting for a :token:`smart_qualid`.
+
+   Or use the :cmd:`Print Implicit` command to see the implicit arguments
    of an object (see :ref:`displaying-implicit-args`).
 
+   .. todo is About always better?  Not sure this should be a note or regular text
```

Comment:
`About` will show not just the implicit arguments, and it will use the `Arguments` syntax for this. `Print Implicit` will use plain textual descriptions. But it might be worth raising the question whether `Print Implicit` could be removed.